packages/web/src/store/application/ui/buy-audio/sagas.ts

```diff
@@ -233,25 +236,51 @@ function* getTransactionFees({
       }
     )
     const connection = yield* call(getSolanaConnection)
+
+    // Calculate fees for transfer transaction (legacy transaction)
     const latestBlockhashResult = yield* call(
       [connection, connection.getLatestBlockhash],
       'finalized'
     )
-    for (const transaction of [swapTransaction, transferTransaction]) {
-      if (transaction) {
-        if (!transaction.recentBlockhash) {
-          transaction.recentBlockhash = latestBlockhashResult.blockhash
-        }
-        if (!transaction.feePayer) {
-          transaction.feePayer = rootAccount
-        }
-        const fee = yield* call(
-          [transaction, transaction.getEstimatedFee],
-          connection
-        )
-        transactionFees += fee ?? 5000 // For some reason, swap transactions don't have fee estimates??
+    if (!transferTransaction.recentBlockhash) {
+      transferTransaction.recentBlockhash = latestBlockhashResult.blockhash
+    }
+    if (!transferTransaction.feePayer) {
+      transferTransaction.feePayer = rootAccount
+    }
+    transactionFees +=
+      (yield* call(
+        [transferTransaction, transferTransaction.getEstimatedFee],
+        connection
+      )) ?? 5000
```

Comment:
nit: (possible followup/todo) Might be time to pull this into a constant somewhere